diff --git a/source/main.py b/source/main.py
index b7c60fe..ee656ee 100644
--- a/source/main.py
+++ b/source/main.py
@@ -39,7 +39,9 @@ app = FastAPI(root_path=root_path)
 VERSION = "4.0.0"
 isDev = os.getenv("NODE_ENV") == "development"
 COMMUNITY_VERSION = True if os.getenv("IS_COMMUNITY_VERSION") == "true" else False
-
+ADDON_ID = os.environ.get("ADDON_ID", "community.aymene69.jackett")
+ADDON_NAME = os.environ.get("ADDON_NAME", "Jackett")
+ADDON_ICON = os.environ.get("ADDON_ICON", "https://i.imgur.com/tVjqEJP.png")
 
 class LogFilterMiddleware:
     def __init__(self, app):
@@ -87,13 +89,13 @@ async def configure(request: Request):
 @app.get("/{params}/manifest.json")
 async def get_manifest():
     return {
-        "id": "community.aymene69.jackett",
-        "icon": "https://i.imgur.com/tVjqEJP.png",
+        "id": ADDON_ID,
+        "icon": ADDON_ICON,
         "version": VERSION,
         "catalogs": [],
         "resources": ["stream"],
         "types": ["movie", "series"],
-        "name": "Jackett" + (" (Dev)" if isDev else ""),
+        "name": ADDON_NAME + (" (Dev)" if isDev else ""),
         "description": "Elevate your Stremio experience with seamless access to Jackett torrent links, effortlessly "
                        "fetching torrents for your selected movies within the Stremio interface.",
         "behaviorHints": {
