diff --git a/comet/utils/general.py b/comet/utils/general.py
index 2b7c320..77cbcdf 100644
--- a/comet/utils/general.py
+++ b/comet/utils/general.py
@@ -345,7 +345,7 @@ async def get_zilean(
                 for result in take_first:
                     object = {
                         "Title": result["filename"],
-                        "InfoHash": result["infoHash"],
+                        "InfoHash": result["info_hash"],
                         "Size": result["filesize"],
                         "Tracker": "DMM",
                     }
