diff --git a/src/main/kotlin/io/skjaere/debridav/PersistenceConfiguration.kt b/src/main/kotlin/io/skjaere/debridav/PersistenceConfiguration.kt
index 0f88c02..e241aeb 100644
--- a/src/main/kotlin/io/skjaere/debridav/PersistenceConfiguration.kt
+++ b/src/main/kotlin/io/skjaere/debridav/PersistenceConfiguration.kt
@@ -30,6 +30,13 @@ class PersistenceConfiguration(
         props.setProperty("dataSource.password", password)
         props["dataSource.logWriter"] = PrintWriter(System.out)
 
+        // Add HikariCP-specific properties to optimize for frequent and bursty database access
+        props.setProperty("maximumPoolSize", "300")
+        props.setProperty("minimumIdle", "5")
+        props.setProperty("connectionTimeout", "5000")
+        props.setProperty("idleTimeout", "30000")
+        props.setProperty("maxLifetime", "1800000")
+
         val config = HikariConfig(props)
         return HikariDataSource(config)
     }
